project-base/storefront/components/Blocks/Product/ProductsList/ProductListItem.tsx

```diff
@@ -26,6 +26,7 @@ type ProductItemProps = {
     isProductInWishlist: boolean;
     toggleProductInComparison: () => void;
     toggleProductInWishlist: () => void;
+    isMiniProduct?: boolean;
```

Comment:
What about naming like `isSimpleCard`? `mini` evoke me literally small product picture or something like that.

Reply:
Yeah, no problem, renamed.